install: add -D example (#7803)
tldr-pages/tldr: pages/common/install.md

- Copy files and apply access/modification times of source to the destination:

`install --preserve-timestamps {{path/to/source}} {{path/to/destination}}`

> - Copy files and create the directories in the path if they don't exist

Reviewer: ```suggestion
- Copy files and create the directories in the path if they don't exist:
```

> - Copy files and create the directories in the path if they don't exist:

Reviewer: ```suggestion
- Copy files and create the directories in the destination directory if they don't exist:
```

Reviewer 2: It's not clear... too many `directories`

Reviewer 2: ```suggestion
- Copy files and create the directories in the destination path if they don't exist:
```

> `install -D {{path/to/source}} {{path/to/destination}}`

Reviewer 3: ```suggestion
`install -D {{path/to/source_file1 path/to/source_file2 ...}} {{path/to/destination_directory}}`
```

Please fix any other code samples to be consistent with this one.

Author: Done
